packages/react-component-library/cypress/selectors/index.ts

```diff
@@ -1,25 +1,25 @@
-import autocompleteE from './AutocompleteE'
+import Autocomplete from './Autocomplete'
 import avatar from './Avatar'
-import checkboxE from './CheckboxE'
+import Checkbox from './Checkbox'
 import contextMenu from './ContextMenu'
 import datePicker from './DatePicker'
 import dismissibleBanner from './DismissibleBanner'
 import form from './form'
-import numberInputE from './NumberInputE'
-import rangeSliderE from './RangeSliderE'
-import selectE from './SelectE'
+import NumberInput from './NumberInput'
+import RangeSlider from './RangeSlider'
+import selectE from './Select'
```

Comment:
Still a `selectE` here:

```suggestion
import select from './Select'
```